build.sbt

```diff
@@ -40,7 +43,33 @@ lazy val `play-slick-root` = (project in file("."))
 lazy val `play-slick` = (project in file("src/core"))
   .enablePlugins(PlayLibrary, Playdoc, MimaPlugin)
   .configs(Docs)
-  .settings(libraryDependencies ++= Dependencies.core)
+  .settings(
+    libraryDependencies ++= Dependencies.core,
+    libraryDependencies ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, _)) =>
+          Seq(
+            "com.typesafe.slick" %% "slick" % "3.4.1",
+            "com.typesafe.slick" %% "slick-hikaricp" % "3.4.1",
+          )
+        case Some((3, _)) =>
+          Seq(
+            "com.typesafe.slick" %% "slick" % "3.4.0-SNAPSHOT",
```

Comment:
This is using a locally published version of Slick with the Scala3 Pull Request. You can find it here: https://github.com/slick/slick/pull/2187